data-science/us-east-1/bedrock-kyb-bda/src/kyb-bda-processor/lambda_function.py

```
@@ -0,0 +1,143 @@
+            clientToken=f"kyb-{timestamp}-{hash(object_key) % 10000}",
+        )
+
+        invocation_arn = response.get("invocationArn")
+        logger.info(f"BDA invocation started with ARN: {invocation_arn}")
+
+        # Store invocation metadata in output bucket for tracking
+        metadata = {
+            "invocation_arn": invocation_arn,
+            "input_s3_uri": input_s3_uri,
+            "output_s3_uri": output_s3_uri,
+            "timestamp": datetime.now(UTC).isoformat(),
```

Ruff (0.12.2)

Undefined name `UTC`

(F821)